CHANGELOG.md

```diff
@@ -40,42 +40,126 @@ Bottom level categories:
 
 ## Unreleased
 
+## 23.0.0 (2024-10-25)
+
+### Themes of this release
+
+This release's theme is one that is likely to repeat for a few releases: convergence with the WebGPU specification! WGPU's design and base functionality are actually determined by two specifications: one for WebGPU, and one for the WebGPU Shading Language.
+
+This may not sound exciting, but let us convince you otherwise! All major web browsers have committed to offering WebGPU in their environment. Even JS runtimes like [Node][nodejs-webgpu-interest] and [Deno][deno_webgpu-crate-manifest] have communities that are very interested in providing WebGPU! WebGPU is slowly [eating the world][eat-the-world-meaning], as it were. 😀 It's really important, then, that WebGPU implementations behave in ways that one would expect across all platforms. For example, if Firefox's WebGPU implementation were to break when running scripts and shaders that worked just fine in Chrome, that would mean sad users for both application authors _and_ browser authors.
+
+[nodejs-webgpu-interest]: https://github.com/orgs/nodejs/discussions/41994
+[deno_webgpu-crate-manifest]: https://github.com/gfx-rs/wgpu/tree/64a61ee5c69569bbb3db03563997e88a229eba17/deno_webgpu#deno_webgpu
+[eat-the-world-meaning]: https://www.quora.com/What-did-Marc-Andreessen-mean-when-he-said-that-software-is-eating-the-world
+
+WGPU also benefits from standard, portable behavior in the same way as web browsers. Because of this behavior, it's generally fairly easy to port over usage of WebGPU in JavaScript to WGPU. It is also what lets WGPU go full circle: WGPU can be an implementation of WebGPU on native targets, but _also_ it can use _other implementations of WebGPU_ as a backend in JavaScript when compiled to WASM. Therefore, the same dynamic applies: if WGPU's own behavior were significantly different, then WGPU and end users would be _sad, sad humans_ as soon as they discover places where their nice apps are breaking, right?
+
+The answer is: yes, we _do_ have sad, sad humans that really want their WGPU code to work _everywhere_. As Firefox and others use WGPU to implement WebGPU, the above example of Firefox diverging from standard is, unfortunately, today's reality. It _mostly_ behaves the same as a standards-compliant WebGPU, but it still doesn't in many important ways. Of particular note is Naga, its implementation of the WebGPU Shader Language. Shaders are pretty a black-and-white point of failure in GPU programming; if they don't compile, then you can't use the rest of the API! And yet, it's extremely easy to run into this:
```

Comment:
typo:
```suggestion
The answer is: yes, we _do_ have sad, sad humans that really want their WGPU code to work _everywhere_. As Firefox and others use WGPU to implement WebGPU, the above example of Firefox diverging from standard is, unfortunately, today's reality. It _mostly_ behaves the same as a standards-compliant WebGPU, but it still doesn't in many important ways. Of particular note is Naga, its implementation of the WebGPU Shader Language. Shaders are pretty much a black-and-white point of failure in GPU programming; if they don't compile, then you can't use the rest of the API! And yet, it's extremely easy to run into this:
```

nit: for longer blocks like this more newlines (which are ignored in rendered markdown) are nicer for reviewing :)

Reply:
RE: typo: fixed with latest force-push to [`78c1c8e5`](https://github.com/gfx-rs/wgpu/compare/06b37fa98f3c8e0b2a38ec6a1f95ad91d809f2d7..78c1c8e5d72d2bff3510e4db56010feaa9b787e6).

Reply:
RE: nit: Counter-nit (🙃): I deliberately coalesced this and existing content to be in single lines to match the style of the rest of the document. I would _really_ like to have soft breaks, because I think it'd make this document easier to deal with WRT diffs and editing. However, this is a big enough style thing that I think we should invoke an opinion from @gfx-rs/wgpu in a non-blocking fashion.

We can always change this formatting later automatically via a tool like `prettier`, so I'm not concerned about this blocking this PR.